tools/c7n_left/c7n_left/providers/terraform/provider.py

```diff
@@ -173,7 +173,7 @@ def resolve_module_ref(self, mod_resource, graph):
 
         call_stack = extract_mod_stack(mod_resource['__tfmeta']['path'])
         ancestor = mod_map[call_stack[0]]
-        ancestor['__tfmeta'].setdefault('call_stack', {})[id(mod_resource)] = call_stack
```

Comment:
Removing `__tfmeta.call_stack` here is intentional?

Reply:
it is, we added it, but never used it anywhere. the refs structure is a simplification of the call_stack to just hold the array of the referencing resources, the call stack itself was just constructed from those referenced resource paths.